test/per_file_coverage.sh

```diff
@@ -18,7 +18,7 @@ declare -a KNOWN_LOW_COVERAGE=(
 "source/common/singleton:95.1"
 "source/common/thread:0.0" # Death tests don't report LCOV
 "source/common/matcher:93.3"
-"source/common/quic:88.4"
+"source/common/quic:87.8"
```

Comment:
This target may be flaky.

Code coverage for source/common/quic is lower than limit of 87.8 (87.7)

https://dev.azure.com/cncf/envoy/_build/results?buildId=76243&view=logs&j=5035ce29-be3c-5fcf-dc50-abb6a265f8b1&t=548c3e59-0417-5971-4f1b-f8545005c291&l=17979